### PR TITLE
fix(ui): render certification on profile page in ul

### LINKED ISSUE #55574: Certifications on profile page should be rendered in a `ul`
## Description

On the user profile page, certifications aren't presented as a group:

```html
<!-- Pseudo code -->
<h2>freeCodeCamp Certifications</h2>
<a>View Responsive Web Design Certification</a>
<a>View JS Certification</a>
<a>View Python Certification</a>
```

<details>
  <summary>Image</summary>
  Screenshot 2024-07-20 at 01 59 27
</details>

Instead, they should be moved to a `ul`, and the list should have a label:

```html
<!-- Pseudo code -->
<h2 id="fcc-certifications">freeCodeCamp Certifications</h2>

<ul aria-labelledby="fcc-certifications">
  <li><a>View Responsive Web Design Certification</a></li>
  <li><a>View JS Certification</a></li>
  <li><a>View Python Certification</a></li>
</ul>
```

## Affected page
The issue is in both the "freeCodeCamp Certifications" and "Legacy Certifications" sections on the profile page (https://www.freecodecamp.org/username or http://localhost:8000/certifieduser).

### PR DESCRIPTION
**Pull request by gagan-bhullar-tech**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #55574

### CODE REVIEW

File: client/src/components/profile/components/certifications.tsx [hunk 1]
```diff
@@ -69,15 +70,17 @@ function Certificates({
 }: CertificationProps): JSX.Element {
   const { t } = useTranslation();
   return (
-    <FullWidthRow className='certifications'>
-      <h2>{t('profile.fcc-certs')}</h2>
+    <FullWidthRow className='profile-certifications'>
+      <h2 id='fcc-certifications'>{t('profile.fcc-certs')}</h2>
```
Comment (shootermv) [inline client/src/components/profile/components/certifications.tsx:74]: i think, it can be better to use css class here (and not id)

Comment (gagan-bhullar-tech) [reply client/src/components/profile/components/certifications.tsx]: this id is not used anywhere. Removed it.

**Comment by gagan-bhullar-tech** [comment]
resolved all the review comments

**Review by huyenltnguyen — APPROVED** [review]

**Comment by lasjorg** [comment]
Do we really need another CSS file just for the UL styles?

I would prefer a couple of tailwind classes (`list-none px-0 m-0`) on the two UL elements.

Also, I would suggest we replace the `<br>` on line 74 with a Spacer component with size medium.

**Comment by lasjorg** [comment]
We already have three patterns, inline CSS, CSS in files, and tailwind.

I don't really see why changing to tailwind whenever possible shouldn't be the choice. The more CSS we add to files, the harder it becomes switching to tailwind later.

As an aside, if we do want it in files, why are we not using CSS Modules for the component styles?

**Comment by gagan-bhullar-tech** [comment]
@huyenltnguyen @lasjorg do we have consensus on the approach for css ?

**Review by moT01 — APPROVED** [review]
LGTM @gagan-bhullar-tech 🎉 Thanks for contributing to freeCodeCamp 🎉 

Since we don't have a standard for our CSS patterns, I think that is outside the scope of this PR and needs to be looked at separately. I am going to merge it 👍

**Pull request merged**